visearch-android/build.gradle

```diff
@@ -40,44 +24,20 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
-}
 
-// Add a new configuration to hold your dependencies
-configurations {
-    myConfig
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
 }
 
+
 dependencies {
-    compile 'com.visenze.datatracking:tracking:0.1.9'
-    compile 'com.squareup.retrofit2:retrofit:2.3.0'
-    compile 'com.squareup.retrofit2:converter-gson:2.3.0'
+    implementation 'com.visenze.datatracking:tracking:0.1.9'
```

Comment:
i think previously Yu Lu revered to compile due to some issues reported by Daniel's customers. Could you double check whether changing to implementation is OK in a demo app?

Reply:
https://github.com/visenze/visearch-sdk-android/commit/f892f4daf1ed71f0fa4248d61a5e722d5af09f81#diff-4035af35a3c9e4da646562bfbf0cdd6ae1c41a6c4cd079b5e9f6dcc472b4e8a0 this is the commit where Yu Lu reverted

Reply:
i believe use compile is the limitation of the bintray repository. as in this pr it is moved to jitpack. the github and jitpack do not have that limitation. i have tried on my side. @yulu

Reply:
ok if you have tested in an app, then it is fine